src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,14 +1,85 @@
 package wolox.training.controllers;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestParam;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
 @Controller
 public class BookController {
+
+    @Autowired
+    private BookRepository bookRepository;
+
     @GetMapping("/greeting")
     public String greeting(@RequestParam(name="name", required=false, defaultValue="World") String name, Model model) {
         model.addAttribute("name", name);
         return "greeting";
     }
+
+    @GetMapping("/create_book")
+    public String create(@RequestParam(name="genre", required=true) String genre, @RequestParam(name="author", required=true) String author, @RequestParam(name="image", required=true)String image,
+        @RequestParam(name="title", required=true) String title, @RequestParam(name="subtitle", required=true) String subtitle, @RequestParam(name="publisher", required=true)String publisher,
+        @RequestParam(name="year", required=true) String year, @RequestParam(name="page", required=true) Integer page, @RequestParam(name="isbn", required=true)String isbn, Model model) {
```

Comment:
Use `@RequestBody` annotation. 
Look at this https://www.baeldung.com/spring-request-response-body

Reply:
okay